### ISSUE TITLE
Remove osxfuse dependency

### ISSUE THREAD

**Issue by dmorn** [jecoz/flexi#10]
Apart from making my macbook crash (https://github.com/osxfuse/osxfuse/issues/713) the fuse driver dependency is unwanted. It makes it difficult to deploy and manage containers in our use-case, cause containers that need the fuse driver obtain this capability by the surrounding environment (fargate does not provide it).

As soon as our images have 9port installed we're taking advantage of that. A combination of `9 srv` and `9p <operation>` might be the solution. We're still deciding wether to CGO 9port or use the executables provided.

TLDR; When this issue is closed we'll drop the fuse dependency 🎉

**Comment by jecoz** [comment]
Furthermore interacting through `9p` instead of `9 mount` is much faster apparently.
